iidm/iidm-modification/src/main/java/com/powsybl/iidm/modification/scalable/ScalingParameters.java

```diff
@@ -11,39 +11,98 @@
 
 import java.util.Objects;
 
+import static com.powsybl.iidm.modification.scalable.ScalingParameters.Priority.ONESHOT;
+import static com.powsybl.iidm.modification.scalable.ScalingParameters.Priority.VOLUME;
+import static com.powsybl.iidm.modification.scalable.ScalingParameters.DistributionMode.REGULAR_DISTRIBUTION;
+import static com.powsybl.iidm.modification.scalable.ScalingParameters.ScalingType.DELTA_P;
+
 /**
  * @author Coline Piloquet <coline.piloquet@rte-france.fr>
  */
 public class ScalingParameters {
 
-    public static final String VERSION = "1.0";
+    public static final String VERSION = "1.1";
+
+    public enum DistributionMode {
+        PROPORTIONAL_TO_TARGETP,
+        PROPORTIONAL_TO_PMAX,
+        PROPORTIONAL_TO_DIFF_PMAX_TARGETP,
+        PROPORTIONAL_TO_DIFF_TARGETP_PMIN,
+        PROPORTIONAL_TO_P0,
+        REGULAR_DISTRIBUTION,
+        STACKING_UP
+    }
+
+    public enum ScalingType {
+        DELTA_P,
+        TARGET_P
+    }
+
+    public enum Priority {
```

Comment:
Volume is understandable, even if I think we can do better. One shot is a good choice, it is really clear. Ventilation is, even in French, really obscur.
- Volume = respect to variation asked.
- Ventilation = respect to distribution of scalables.
Right?

Reply:
changed